client/java-spring-boot-autoconfigure/src/main/java/com/linecorp/centraldogma/client/spring/CentralDogmaClientAutoConfiguration.java

```diff
@@ -87,13 +89,15 @@ public CentralDogma dogmaClient(
             Environment env,
             CentralDogmaSettings settings,
             @ForCentralDogma ClientFactory clientFactory,
-            Optional<ArmeriaClientConfigurator> armeriaClientConfigurator) throws UnknownHostException {
+            Optional<ArmeriaClientConfigurator> armeriaClientConfigurator,
+            Optional<Consumer<DnsAddressEndpointGroupBuilder>> dnsAddressEndpointGroupCustomizer) throws UnknownHostException {
```

Comment:
Don't we need a dedicated interface if we choose to approach in the same way as `ArmeriaClientConfigurator` such as:
```java
public interface DnsAddressEndpointGroupConfigurator {
    void configure(DnsAddressEndpointGroupBuilder builder);
}

Reply:
okay, it works well without but it could be useful. Let me add it.